Babylon/commands/azure/staticwebapp/services/swa_api_svc.py

```diff
@@ -36,6 +36,7 @@ def create(self, webapp_name: str, details: str):
             f"providers/Microsoft.Web/staticSites/{webapp_name}?api-version=2022-03-01")
         response = oauth_request(route, self.azure_token, type="PUT", data=details)
         if response is None:
+            logger.error(f"[webapp] Failed to launch of webapp {webapp_name} in resource group {resource_group_name}")
```

Comment:
C'est plutôt mieux ici :
`logger.error(f"[webapp] Failed to create  webapp {webapp_name} in resource group {resource_group_name}")`